.github/workflows/node.js.yml

```diff
@@ -16,8 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x]
-        # node-version: [12.x, 14.x, 16.x]
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
Angular 9 is only compatible with node v10.13.x or v12.11.x or later minor versions, so node 14 and 16 are prone to fail. In my opinion it isn't necessary to test incompatible node versions

Reply:
but node 14.x seems to be ok as well!?

Reply:
of course it may work. but it's not officially supported. maybe your old mercedes also runs on salad oil, doesn't mean it's intended by the manufacturer ;-)

Reply:
See https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3

Reply:
As long as I can verify that this dos not spoil the engine that would be ok for me, this is what tests do: Verify (prove) that it **is** working, although the manufacturer, does not guarantee it (assuming have a test base proving that). 
On the other side: If **we** don't have use or need for running the product in that combination it makes little sense proving that it would.